packages/grafana-schema/src/veneer/dashboard.types.ts

```diff
@@ -10,24 +10,131 @@ export interface Panel<TOptions = Record<string, unknown>, TCustomFieldConfig =
   fieldConfig: FieldConfigSource<TCustomFieldConfig>;
 }
 
-export enum VariableHide {
-  dontHide,
-  hideLabel,
-  hideVariable,
+export interface BaseVariableModel extends Omit<raw.BaseVariableModel, 'rootStateKey' | 'error' | 'description'> {
+  // Overrides nullable properties because CUE doesn't support null values
+  rootStateKey: string | null;
+  error: any | null;
+  description: string | null;
+}
+
+export interface VariableWithOptions extends Omit<raw.VariableWithOptions, 'rootStateKey' | 'error' | 'description'> {
+  // Overrides nullable properties because CUE doesn't support null values
+  rootStateKey: string | null;
+  error: any | null;
+  description: string | null;
+}
+
+export interface VariableWithMultiSupport
+  extends Omit<raw.VariableWithMultiSupport, 'rootStateKey' | 'error' | 'description' | 'allValue'> {
+  // Overrides nullable properties because CUE doesn't support null values
+  rootStateKey: string | null;
+  error: any | null;
```

Comment:
What is the error here?  would it get saved in dashboard json?

is there anything better than `any ` we can use?  (i know that is our current model)